pkg/adapter/mtping/runner_test.go

```diff
@@ -206,6 +204,45 @@ func TestStartStopCron(t *testing.T) {
 
 }
 
+func TestStartStopCronDelayWait(t *testing.T) {
+	tn := time.Now()
+	seconds := tn.Second()
+	if seconds > 57 {
```

Comment:
57 should be a constant, otherwise it's hard to guess your intention.

Reply:
I put a constant